bin/docker_rsync.py

```diff
@@ -39,28 +40,31 @@ def run_rsync(volumes, from_path, to_path, relative):
 def get_volume_args(settings, local_volume, volume_mode):
     mounted_volume = join("/", local_volume)
     return {
-        settings.known_hosts_path: {"bind": "/root/.ssh/known_hosts", "mode": "rw"},
-        settings.ssh_key_path: {"bind": "/root/.ssh/id_rsa", "mode": "ro"},
+        "bb8_ssh": {"bind": "/root/.ssh", "mode": "rw"},
         local_volume: {"bind": mounted_volume, "mode": volume_mode}
     }
 
 
+def get_remote_dir(starport):
+    return "bb8@{}:{}".format(starport["addr"], starport["backup_location"])
```

Comment:
Should we pass bb8 in as `startport["user"]` and make this configurable too?  New ticket?

Reply:
Fixed